routing/localchans/manager.go

```diff
@@ -155,7 +172,58 @@ func (r *Manager) UpdatePolicy(newSchema routing.ChannelPolicy,
 					"not yet confirmed",
 				))
 
+		case createMissingEdge:
+			// If the edge was not found, but the channel is found,
+			// that means the edge is missing in the graph database
+			// and should be recreated. The edge and policy are
+			// created in-memory and will be added to the graph in
+			// the PropagateChanPolicyUpdate call below.
+			log.Warnf("Missing edge for active channel (%s) "+
+				"during policy update. Recreating edge with "+
+				"default policy.",
+				channel.FundingOutpoint.String())
+
+			info, edge, err := r.createEdge(channel)
+			if err != nil {
+				log.Errorf("Failed to recreate missing edge "+
+					"for channel (%s): %v",
+					channel.FundingOutpoint.String(), err)
+
+				f := lnrpc.UpdateFailure_UPDATE_FAILURE_UNKNOWN
+				failedUpdates = append(failedUpdates,
+					makeFailureItem(chanPoint, f,
+						"could not update policies",
+					))
+			}
+
+			// Insert the edge into the database, to avoid edge not
+			// found errors during policy update propagation.
+			err = r.AddEdge(info)
+			switch {
+			case routing.IsError(err, routing.ErrIgnored):
```

Comment:
I think we should error out here although the error is `ErrIgnored` that should never happen and if it does why are we adding the edge in the first place, so I think erroring out its better.

Reply:
OK makes sense

Reply:
Done